include/d/actor/d_a_ship.h

```diff
@@ -74,7 +77,7 @@ class daShip_c : public fopAc_ac_c {
     void onSceneChange() {}
     void onShortHitFlg() {}
     void onStateFlg(daSHIP_SFLG) {}
-    void onTornadoFlg(unsigned long) {}
+    void onTornadoFlg(int tornadoID) { mTornadoID = tornadoID; }
```

Comment:
```suggestion
    void onTornadoFlg(u32 tornadoID) { mTornadoID = tornadoID; }
```
We know this inline takes a `u32` (aka unsigned long) from the debug maps. You shouldn't change function parameter types, only return types and field types.